pom.xml

```diff
@@ -50,52 +50,64 @@
 
     <properties>
         <kotlin.compiler.jvmTarget>1.8</kotlin.compiler.jvmTarget>
-        <dokka.version>0.9.17</dokka.version>
-        <kotlin.version>1.3.72</kotlin.version>
-        <kotlinx.coroutines.version>1.3.7</kotlinx.coroutines.version>
-        <junit.version>5.0.2</junit.version>
         <project.build.sourceEncoding>UTF-8</project.build.sourceEncoding>
-        <source.plugin.version>3.0.1</source.plugin.version>
-        <javadoc.plugin.version>3.0.0</javadoc.plugin.version>
-        <spring.webflux.version>5.2.6.RELEASE</spring.webflux.version>
+
+        <kotlin.version>1.4.10</kotlin.version>
+        <kotlinx.coroutines.version>1.3.9</kotlinx.coroutines.version>
+        <spring-boot.version>2.3.5.RELEASE</spring-boot.version>
+        <swagger-request-validator.version>2.11.0</swagger-request-validator.version>
     </properties>
 
     <pluginRepositories>
+        <!-- Required for Dokka -->
         <pluginRepository>
             <id>jcenter</id>
             <name>JCenter</name>
             <url>https://jcenter.bintray.com/</url>
         </pluginRepository>
     </pluginRepositories>
 
+    <dependencyManagement>
+        <dependencies>
+            <dependency>
+                <groupId>org.jetbrains.kotlin</groupId>
+                <artifactId>kotlin-bom</artifactId>
+                <version>${kotlin.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+            <dependency>
+                <groupId>org.springframework.boot</groupId>
+                <artifactId>spring-boot-dependencies</artifactId>
+                <version>${spring-boot.version}</version>
+                <type>pom</type>
+                <scope>import</scope>
+            </dependency>
+        </dependencies>
+    </dependencyManagement>
+
     <dependencies>
-        <dependency>
-            <groupId>com.fasterxml.jackson.module</groupId>
-            <artifactId>jackson-module-kotlin</artifactId>
-            <version>2.11.0</version>
-            <scope>compile</scope>
-        </dependency>
         <dependency>
             <groupId>com.atlassian.oai</groupId>
             <artifactId>swagger-request-validator-core</artifactId>
-            <version>2.10.0</version>
-            <scope>compile</scope>
+            <version>${swagger-request-validator.version}</version>
         </dependency>
         <dependency>
             <groupId>org.springframework</groupId>
             <artifactId>spring-webflux</artifactId>
-            <version>${spring.webflux.version}</version>
-            <scope>compile</scope>
         </dependency>
+        <dependency>
+            <groupId>com.fasterxml.jackson.module</groupId>
+            <artifactId>jackson-module-kotlin</artifactId>
```

Comment:
curious about the decision to remove a bunch of the versions? seems beneficial to specify an explicit version

Reply:
Since we target the Spring Boot universe, the only version specified is that of the Spring Boot Release (used in the BOM import). Everything else that we don't know better should be taken from there.

Relying on Pivotal seems to be more effective than tuning yourself, since it is related to compile dependencies only.
Especially, if you want to use this library we should not downgrade versions from those specified by the spring-boot dependencies. If there is a good reason to differ from versions specified in Spring Boot BOM, it should be overwritten, but for now I see no reason for that.

Provided dependencies are (among many others):
- Spring Webflux
- Jackson
- JUnit 5
- Kotlin -> but here we should not rely on SpringBoot, see comment below

I separated Kotlin out of it, because they release more often than 1-2 times a year. 1.4.10 is not in current spring boot yet... For the maven plugins, I specified all versions in plugin management... 

Another idea by the way would be to specified all dependencies besides Atlassian library as provided and cut the dependency trail for the users of the library. I think this is even a better than write a version inside and update it all the time.

Reply:
Makes sense. 👍